### ISSUE TITLE
[[Idea]] Leverage Zotero for storage and metadata handling of PDFs

### ISSUE THREAD

**Issue by pcuellar** [elias-sundqvist/obsidian-annotator#22]
Especially for those writing academic material, it will be very helpful to connect this super annotating tool with a reference manager. You can leverage metadata, notes and other data pieces from Zotero native or its famous plugins (e.g. Zotfile).

**Comment by tyathom** [comment]
+1

**Comment by ryanwwest** [comment]
I agree - see https://github.com/elias-sundqvist/obsidian-annotator/issues/139 for some ideas on eventual two-way annotations syncing.

**Comment by ryanwwest** [comment]
As I think about it more, perhaps this plugin could just utilize https://github.com/mgmeyers/obsidian-zotero-integration in some way, as they've already figured out a lot of this. In fact, you  might be able to import annotations from Zotero PDFs into the format needed for reading annotations in the Hypothes.is reader in this plugin (that plugin has import templates), but it might not completely work with the `href` JSON field required by this plugin.
